Description: Don't turn format warnings into errors
 Works around a build failure on s390x with gcc versions higher than gcc-9.
Author: Gunnar Hjalmarsson <gunnarhj@ubuntu.com>

This is the problematic warning:

In file included from display-server.h:19,
                 from x-server.h:16,
                 from x-server.c:16:
x-server.c: In function ‘x_server_start’:
logger.h:54:5: error: ‘%s’ directive argument is null [-Werror=format-overflow=]
   54 |     logger_log (LOGGER (self), G_LOG_LEVEL_DEBUG, __VA_ARGS__)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x-server.c:131:45: note: format string is defined here
  131 |     l_debug (server, "Connecting to XServer %s", x_server_get_address (server));
      |                                             ^~

Reported upstream: https://github.com/canonical/lightdm/issues/188

--- a/configure.ac
+++ b/configure.ac
@@ -29,7 +29,6 @@
                  -Werror=pointer-arith \
                  -Werror=init-self \
                  -Werror=format-security \
-                 -Werror=format=2 \
                  -Werror=missing-include-dirs"
     WARN_CXXFLAGS="-Wall"
 fi
